extensions/goose/src/kube-template.spec.ts

```
@@ -43,6 +43,36 @@ instructions: |
+    kortex: {
+      version: '1.0.0',
+    },
+  },
+  {
+    testName: 'parameterized-recipe',
+    job: { name: 'job-1' },
+    namespace: 'ns1',
+    recipe: {
+      flowId: 'demo-flow-id',
+      name: 'echo',
+      content: `dummy`,
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)